client/packages/programs/src/JsonForms/common/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import { RegexUtils } from '@common/utils';
+import {
+  LayoutProps,
+  uiTypeIs,
+  rankWith,
+  UISchemaElement,
+} from '@jsonforms/core';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import { Typography } from '@mui/material';
+import React from 'react';
+
+export const headerTester = rankWith(10, uiTypeIs('Header'));
+
+type HeaderProps = LayoutProps & {
+  uischema: UISchemaElement & {
+    header?: string;
+    subheader?: string;
```

Comment:
we use to put these custom field into the options object and then validate the options using zod...